weedcoco/schema/compatible-coco.yaml

```diff
@@ -18,4 +18,4 @@ properties:
   categories:
     type: array
     items:
-      $ref: https://weedid.sydney.edu.au/schema/Category.json#/
```

Comment:
The thing that defined "compatible-coco" was that it had the weedcoco category requirements. If we're getting rid of the category requirements, we should call this format "coco", not "compatible-coco".

Reply:
Do we want to keep both coco and compatible-coco? or just rename it to coco instead.